.azure-pipelines/warden.yml

```diff
@@ -24,6 +24,34 @@ jobs:
       python setup.py sdist -d $(Build.ArtifactStagingDirectory) --format zip
     displayName: 'Build Package'
 
+  - task: CopyFiles@2
+    displayName: 'Copy files for github release'
+    inputs:
+      SourceFolder: '$(Build.SourcesDirectory)'
+      Contents: 'scripts/*/verify_agent_os.py'
```

Comment:
I'd prefer not to list all the script explicitly in the yml files so I suggest just copying all the scripts.

Reply:
You mean copy all files in the script folder?

Reply:
Yes lets start by just copying all scripts in the scripts folder and if we feel that needs to be scoped then we can setup a directory structure in the repo to help with the divide instead of listing the scripts one-by-one here in the yml.

Reply:
So all the python scripts excluding the readmes?